basics-assignment-2-start/src/app/app.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl:'./app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  values = ''
```

Comment:
The best practice is always to define the type of the variable and format the file